bpf/bpf_sock.c

```diff
@@ -286,6 +286,49 @@ int sock4_xlate(struct bpf_sock_addr *ctx)
 	return SYS_PROCEED;
 }
 
+#ifdef ENABLE_NODEPORT
+__section("bind-sock4")
```

Comment:
Needs the usual ifdef v4/v6.

Reply:
Overall looks great to me!

Reply:
> Needs the usual ifdef v4/v6.

Not sure what you mean here - the `ifdef ENABLE_IPV4/6` are here (afaict):
https://github.com/cilium/cilium/blob/094c66962dd33c0a0c0b6a88ebe6765c45fcbb8d/bpf/bpf_sock.c#L114
https://github.com/cilium/cilium/blob/094c66962dd33c0a0c0b6a88ebe6765c45fcbb8d/bpf/bpf_sock.c#L421

Reply:
Sigh, yes, missed that.